WordBreak.py

```diff
@@ -0,0 +1,28 @@
+'''
+Word Break
+Given an input string and a dictionary of words,
+find out if the input string can be segmented into a space-separated sequence of dictionary words.
+For example, consider the following dictionary:
+{ pear, salmon, foot, prints, footprints, leave, you, sun, girl, enjoy },

+Examples:
Given the string “youenjoy”, 
+Output: True (The string can be segmented as “you enjoy”)

+Input: “youleavefootprints”,
+Output: True (The string can be segmented as “you leave footprints” or “you leave foot prints”)
+Input:salmonenjoyapples
+Output: False
+'''
+class Solution():
+    def wordBreak(self, s, dictionary):
+        if s == "" or dictionary == {}:
+            return False
+        s = s.lower()
+        for word in dictionary:
+            word = word.lower()
+            if word in s:
+                s = s.replace(word,"")
```

Comment:
Hey thanks for pointing that out. I fixed the code to take care of test cases like the one you mentioned!